commit 12726cda009a855073b9bedbdce57b6ce7763da2
Author: Peter Dimov <pdimov@pdimov.com>
Date:   Sat Dec 2 04:10:46 2017 +0200

    Add #include <boost/next_prior.hpp>; no longer in utility.hpp

diff --git a/include/boost/lockfree/spsc_queue.hpp b/include/boost/lockfree/spsc_queue.hpp
index 7903d2c..c7f2953 100644
--- a/libs/lockfree/include/boost/lockfree/spsc_queue.hpp
+++ b/libs/lockfree/include/boost/lockfree/spsc_queue.hpp
@@ -17,6 +17,7 @@
 #include <boost/assert.hpp>
 #include <boost/static_assert.hpp>
 #include <boost/utility.hpp>
+#include <boost/next_prior.hpp>
 #include <boost/utility/enable_if.hpp>
 #include <boost/config.hpp> // for BOOST_LIKELY
 
